docs/rendering/modelextensions/facedata.md

```diff
@@ -0,0 +1,109 @@
+Face Data
+=========
+
+In a vanilla "elements" model, additional data about an element's faces can be specified at either the element level or the face level. Faces which do not specify their own face data will fall back to the element's face data or a default if no face data is specified at the element level.
+
+To use this extension for a generated item model, the model must be loaded through the `forge:item_layers` model loader due to the vanilla item model generator not being extended to read this additional data.
+
+All values of the face data are optional.
+
+Elements Model
+--------------
+
+In vanilla "elements" models, the face data applies to the face it is specified in or all faces of the element it is specified in which don't have their own face data.
+
+The additional data can be specified in the two ways shown in this example:
+```js
+{
+  "elements": [
+    {
+      "forge_data": {
+        "color": "0xFFFF0000",
+          "block_light": 15,
+          "sky_light": 15,
+          "ambient_occlusion": false
+      },
+      "faces": {
+        "north": {
+          "forge_data": {
+            "color": "0xFFFF0000",
+            "block_light": 15,
+            "sky_light": 15,
+            "ambient_occlusion": false
+          },
```

Comment:
Are the face and top model specification an overwrite or is it merged from the parent if a field is not defined?

Reply:
Added a comment detailing the override behaviour